ExampleCode/Java/needleDrop/src/needleDrop/NeedleClass.java

```diff
@@ -0,0 +1,57 @@
+package needleDrop;
+import java.util.Random;
+
+
+public class NeedleClass {
+	
+	//Creates a Needle class with a variable for the number of throws and hits
+	
+	   private Random generator;
```

Comment:
Indentation issue.  The comment and this line of code do not line up.  Also, remove the space between them.